packages/auth0-ai/auth0_ai/authorizers/async_authorization/async_authorizer_base.py

```diff
@@ -81,13 +81,13 @@ def __init__(self, params: CIBAAuthorizerParams[ToolInput], auth0: Auth0ClientPa
         self.params = params
 
         # TODO: consider moving this to Auth0AI classes
-        ciba_store = SubStore(params["store"] if "store" in params else InMemoryStore()).create_sub_store("AUTH0_AI_CIBA")
+        async_auth_store = SubStore(params["store"] if "store" in params else InMemoryStore()).create_sub_store("AUTH0_AI_CIBA")
```

Comment:
We might as well swap the store key `AUTH0_AI_CIBA`, or any reason to keep it as is? Is it to minimize disruption of an already deployed agent app that uses this key?

Reply:
6e84bc4a79e094a12a0815a9f04620e66aced83e